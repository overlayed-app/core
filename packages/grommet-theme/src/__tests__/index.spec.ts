import theme from '../'

describe('grommet-theme', () => {
  it('should match snapshot', () => {
    expect(theme).toMatchSnapshot('theme-snapshot')
  })
})
