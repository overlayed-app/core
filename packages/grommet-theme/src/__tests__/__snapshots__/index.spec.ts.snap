// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`grommet-theme should match snapshot: theme-snapshot 1`] = `
Object {
  "accordion": Object {
    "border": Object {
      "color": "border",
      "side": "bottom",
    },
    "heading": Object {
      "level": "4",
    },
    "icons": Object {},
  },
  "anchor": Object {
    "color": "control",
    "fontWeight": 600,
    "hover": Object {
      "textDecoration": "underline",
    },
    "textDecoration": "none",
  },
  "box": Object {
    "responsiveBreakpoint": "small",
  },
  "button": Object {
    "border": Object {
      "radius": "18px",
      "width": "2px",
    },
    "padding": Object {
      "horizontal": "22px",
      "vertical": "4px",
    },
    "primary": Object {},
  },
  "calendar": Object {
    "heading": Object {
      "level": "4",
    },
    "icons": Object {
      "small": Object {},
    },
    "large": Object {
      "daySize": "109.71428571428571px",
      "fontSize": "30px",
      "lineHeight": 1.11,
      "slideDuration": "0.8s",
    },
    "medium": Object {
      "daySize": "54.857142857142854px",
      "fontSize": "18px",
      "lineHeight": 1.45,
      "slideDuration": "0.5s",
    },
    "small": Object {
      "daySize": "27.428571428571427px",
      "fontSize": "14px",
      "lineHeight": 1.375,
      "slideDuration": "0.2s",
    },
  },
  "carousel": Object {
    "animation": Object {
      "duration": 1000,
    },
    "disabled": Object {
      "icons": Object {},
    },
    "icons": Object {},
  },
  "chart": Object {
    "color": "accent-1",
  },
  "checkBox": Object {
    "border": Object {
      "color": Object {
        "dark": "rgba(255, 255, 255, 0.5)",
        "light": "rgba(0, 0, 0, 0.15)",
      },
      "width": "2px",
    },
    "check": Object {
      "radius": "4px",
      "thickness": "4px",
    },
    "hover": Object {
      "border": Object {
        "color": Object {
          "dark": "white",
          "light": "black",
        },
      },
    },
    "icon": Object {},
    "icons": Object {},
    "size": "24px",
    "toggle": Object {
      "color": Object {
        "dark": "#d9d9d9",
        "light": "#d9d9d9",
      },
      "knob": Object {},
      "radius": "24px",
      "size": "48px",
    },
  },
  "clock": Object {
    "analog": Object {
      "hour": Object {
        "color": Object {
          "dark": "light-2",
          "light": "dark-3",
        },
        "shape": "round",
        "size": "24px",
        "width": "8px",
      },
      "minute": Object {
        "color": Object {
          "dark": "light-4",
          "light": "dark-3",
        },
        "shape": "round",
        "size": "12px",
        "width": "4px",
      },
      "second": Object {
        "color": Object {
          "dark": "accent-1",
          "light": "accent-1",
        },
        "shape": "round",
        "size": "9px",
        "width": "3px",
      },
      "size": Object {
        "huge": "288px",
        "large": "144px",
        "medium": "96px",
        "small": "72px",
        "xlarge": "216px",
      },
    },
    "digital": Object {
      "text": Object {
        "large": Object {
          "height": 1.167,
          "size": "22px",
        },
        "medium": Object {
          "height": 1.375,
          "size": "18px",
        },
        "small": Object {
          "height": 1.43,
          "size": "14px",
        },
        "xlarge": Object {
          "height": 1.1875,
          "size": "26px",
        },
        "xsmall": Object {
          "height": 1.5,
          "size": "10px",
        },
        "xxlarge": Object {
          "height": 1.125,
          "size": "34px",
        },
      },
    },
  },
  "collapsible": Object {
    "baseline": 500,
    "minSpeed": 200,
  },
  "dataTable": Object {
    "groupEnd": Object {
      "border": Object {
        "side": "bottom",
        "size": "xsmall",
      },
    },
    "groupHeader": Object {
      "background": Object {
        "dark": "dark-2",
        "light": "light-2",
      },
      "border": Object {
        "side": "bottom",
        "size": "xsmall",
      },
      "pad": Object {
        "horizontal": "small",
        "vertical": "xsmall",
      },
    },
    "header": Object {},
    "icons": Object {},
    "primary": Object {
      "weight": "bold",
    },
    "resize": Object {
      "border": Object {
        "color": "border",
        "side": "right",
      },
    },
  },
  "diagram": Object {
    "line": Object {
      "color": "accent-1",
    },
  },
  "formField": Object {
    "border": Object {
      "color": "border",
      "error": Object {
        "color": Object {
          "dark": "white",
          "light": "status-critical",
        },
      },
      "position": "inner",
      "side": "bottom",
    },
    "content": Object {
      "pad": Object {
        "bottom": "small",
        "horizontal": "small",
      },
    },
    "error": Object {
      "color": Object {
        "dark": "status-critical",
        "light": "status-critical",
      },
      "margin": Object {
        "horizontal": "small",
        "vertical": "xsmall",
      },
    },
    "help": Object {
      "color": Object {
        "dark": "dark-3",
        "light": "dark-3",
      },
      "margin": Object {
        "left": "small",
      },
    },
    "label": Object {
      "margin": Object {
        "horizontal": "small",
        "vertical": "xsmall",
      },
    },
    "margin": Object {
      "bottom": "small",
    },
  },
  "global": Object {
    "active": Object {
      "background": Object {
        "color": "active",
        "opacity": "medium",
      },
      "color": Object {
        "dark": "white",
        "light": "black",
      },
    },
    "animation": Object {
      "duration": "1s",
      "jiggle": Object {
        "duration": "0.1s",
      },
    },
    "borderSize": Object {
      "large": "12px",
      "medium": "4px",
      "small": "2px",
      "xlarge": "24px",
      "xsmall": "1px",
    },
    "breakpoints": Object {
      "large": Object {},
      "medium": Object {
        "value": 1536,
      },
      "small": Object {
        "borderSize": Object {
          "large": "6px",
          "medium": "4px",
          "small": "2px",
          "xlarge": "12px",
          "xsmall": "1px",
        },
        "edgeSize": Object {
          "hair": "1px",
          "large": "24px",
          "medium": "12px",
          "none": "0px",
          "small": "6px",
          "xlarge": "48px",
          "xsmall": "3px",
          "xxsmall": "2px",
        },
        "size": Object {
          "full": "100%",
          "large": "384px",
          "medium": "192px",
          "small": "96px",
          "xlarge": "768px",
          "xsmall": "48px",
          "xxsmall": "24px",
        },
        "value": 768,
      },
    },
    "colors": Object {
      "accent-1": "#FFB2B2",
      "accent-2": "#129EB7",
      "accent-3": "#60EBE1",
      "accent-4": "#FFCA58",
      "active": "rgba(102,102,102,0.5)",
      "background": "#111111",
      "black": "#000000",
      "border": Object {
        "dark": "rgba(255,255,255,0.33)",
        "light": "rgba(0,0,0,0.33)",
      },
      "brand": "#640FC4",
      "control": Object {
        "dark": "#FFCA58",
        "light": "#403216",
      },
      "dark-1": "#333333",
      "dark-2": "#555555",
      "dark-3": "#777777",
      "dark-4": "#999999",
      "dark-5": "#999999",
      "dark-6": "#999999",
      "focus": "#FFCA58",
      "icon": Object {
        "0": "#",
        "1": "6",
        "2": "6",
        "3": "6",
        "4": "6",
        "5": "6",
        "6": "6",
        "dark": "#f8f8f8",
        "light": "#666666",
      },
      "light-1": "#F8F8F8",
      "light-2": "#F2F2F2",
      "light-3": "#EDEDED",
      "light-4": "#DADADA",
      "light-5": "#DADADA",
      "light-6": "#DADADA",
      "neutral-1": "#EB6060",
      "neutral-2": "#01C781",
      "neutral-3": "#6095EB",
      "neutral-4": "#FFB200",
      "placeholder": "#AAAAAA",
      "selected": "brand",
      "status-critical": "#FF3333",
      "status-disabled": "#a8a8a8",
      "status-error": "#FF3333",
      "status-ok": "#7DD892",
      "status-unknown": "#a8a8a8",
      "status-warning": "#F7E464",
      "text": Object {
        "dark": "#eeeeee",
        "light": "#444444",
      },
      "white": "#FFFFFF",
    },
    "control": Object {
      "border": Object {
        "color": "border",
        "radius": "4px",
        "width": "1px",
      },
      "disabled": Object {
        "opacity": 0.3,
      },
    },
    "debounceDelay": 300,
    "deviceBreakpoints": Object {
      "computer": "large",
      "phone": "small",
      "tablet": "medium",
    },
    "drop": Object {
      "background": "#333333",
      "border": Object {
        "radius": "0px",
      },
      "shadowSize": "small",
      "zIndex": "20",
    },
    "edgeSize": Object {
      "hair": "1px",
      "large": "48px",
      "medium": "24px",
      "none": "0px",
      "responsiveBreakpoint": "small",
      "small": "12px",
      "xlarge": "96px",
      "xsmall": "6px",
      "xxsmall": "3px",
    },
    "elevation": Object {
      "dark": Object {
        "large": "0px 8px 16px rgba(255, 255, 255, 0.40)",
        "medium": "0px 6px 8px rgba(255, 255, 255, 0.40)",
        "none": "none",
        "small": "0px 4px 4px rgba(255, 255, 255, 0.40)",
        "xlarge": "0px 12px 24px rgba(255, 255, 255, 0.40)",
        "xsmall": "0px 2px 2px rgba(255, 255, 255, 0.40)",
      },
      "light": Object {
        "large": "0px 8px 16px rgba(0, 0, 0, 0.20)",
        "medium": "0px 4px 8px rgba(0, 0, 0, 0.20)",
        "none": "none",
        "small": "0px 2px 4px rgba(0, 0, 0, 0.20)",
        "xlarge": "0px 12px 24px rgba(0, 0, 0, 0.20)",
        "xsmall": "0px 1px 2px rgba(0, 0, 0, 0.20)",
      },
    },
    "focus": Object {
      "border": Object {
        "color": Array [
          null,
          ";",
        ],
        "width": "2px",
      },
    },
    "font": Object {
      "family": "\\"Poppins\\", sans-serif",
      "height": "24px",
      "maxWidth": "432px",
      "size": "18px",
    },
    "hover": Object {
      "background": Object {
        "color": "active",
        "opacity": "medium",
      },
      "color": Object {
        "dark": "white",
        "light": "black",
      },
    },
    "input": Object {
      "padding": "12px",
      "weight": 700,
    },
    "opacity": Object {
      "medium": 0.4,
      "strong": 0.8,
      "weak": 0.1,
    },
    "selected": Object {
      "background": "selected",
      "color": "white",
    },
    "size": Object {
      "full": "100%",
      "large": "768px",
      "medium": "384px",
      "small": "192px",
      "xlarge": "1152px",
      "xsmall": "96px",
      "xxlarge": "1536px",
      "xxsmall": "48px",
    },
    "spacing": "24px",
  },
  "grommet": Object {},
  "heading": Object {
    "font": Object {},
    "level": Object {
      "1": Object {
        "font": Object {},
        "large": Object {
          "height": "88px",
          "maxWidth": "1968px",
          "size": "82px",
        },
        "medium": Object {
          "height": "56px",
          "maxWidth": "1200px",
          "size": "50px",
        },
        "small": Object {
          "height": "40px",
          "maxWidth": "816px",
          "size": "34px",
        },
        "xlarge": Object {
          "height": "120px",
          "maxWidth": "2736px",
          "size": "114px",
        },
      },
      "2": Object {
        "font": Object {},
        "large": Object {
          "height": "56px",
          "maxWidth": "1200px",
          "size": "50px",
        },
        "medium": Object {
          "height": "40px",
          "maxWidth": "816px",
          "size": "34px",
        },
        "small": Object {
          "height": "32px",
          "maxWidth": "624px",
          "size": "26px",
        },
        "xlarge": Object {
          "height": "72px",
          "maxWidth": "1584px",
          "size": "66px",
        },
      },
      "3": Object {
        "font": Object {},
        "large": Object {
          "height": "40px",
          "maxWidth": "816px",
          "size": "34px",
        },
        "medium": Object {
          "height": "32px",
          "maxWidth": "624px",
          "size": "26px",
        },
        "small": Object {
          "height": "28px",
          "maxWidth": "528px",
          "size": "22px",
        },
        "xlarge": Object {
          "height": "48px",
          "maxWidth": "1008px",
          "size": "42px",
        },
      },
      "4": Object {
        "font": Object {},
        "large": Object {
          "height": "24px",
          "maxWidth": "432px",
          "size": "18px",
        },
        "medium": Object {
          "height": "24px",
          "maxWidth": "432px",
          "size": "18px",
        },
        "small": Object {
          "height": "24px",
          "maxWidth": "432px",
          "size": "18px",
        },
        "xlarge": Object {
          "height": "24px",
          "maxWidth": "432px",
          "size": "18px",
        },
      },
      "5": Object {
        "font": Object {},
        "large": Object {
          "height": "22px",
          "maxWidth": "384px",
          "size": "16px",
        },
        "medium": Object {
          "height": "22px",
          "maxWidth": "384px",
          "size": "16px",
        },
        "small": Object {
          "height": "22px",
          "maxWidth": "384px",
          "size": "16px",
        },
        "xlarge": Object {
          "height": "22px",
          "maxWidth": "384px",
          "size": "16px",
        },
      },
      "6": Object {
        "font": Object {},
        "large": Object {
          "height": "20px",
          "maxWidth": "336px",
          "size": "14px",
        },
        "medium": Object {
          "height": "20px",
          "maxWidth": "336px",
          "size": "14px",
        },
        "small": Object {
          "height": "20px",
          "maxWidth": "336px",
          "size": "14px",
        },
        "xlarge": Object {
          "height": "20px",
          "maxWidth": "336px",
          "size": "14px",
        },
      },
    },
    "responsiveBreakpoint": "small",
    "weight": 600,
  },
  "icon": Object {
    "size": Object {
      "large": "48px",
      "medium": "24px",
      "small": "12px",
      "xlarge": "96px",
    },
  },
  "layer": Object {
    "background": "#111111",
    "border": Object {
      "radius": "4px",
    },
    "container": Object {
      "zIndex": "15",
    },
    "overlay": Object {
      "background": "rgba(48,48,48,0.5)",
    },
    "responsiveBreakpoint": "small",
    "zIndex": "10",
  },
  "list": Object {
    "item": Object {
      "border": "horizontal",
      "pad": Object {
        "horizontal": "medium",
        "vertical": "small",
      },
    },
  },
  "maskedInput": Object {},
  "menu": Object {
    "icons": Object {},
  },
  "meter": Object {
    "color": "accent-1",
  },
  "paragraph": Object {
    "large": Object {
      "height": "28px",
      "maxWidth": "528px",
      "size": "22px",
    },
    "medium": Object {
      "height": "24px",
      "maxWidth": "432px",
      "size": "18px",
    },
    "small": Object {
      "height": "20px",
      "maxWidth": "336px",
      "size": "14px",
    },
    "xlarge": Object {
      "height": "32px",
      "maxWidth": "624px",
      "size": "26px",
    },
    "xxlarge": Object {
      "height": "40px",
      "maxWidth": "816px",
      "size": "34px",
    },
  },
  "radioButton": Object {
    "border": Object {
      "color": Object {
        "dark": "rgba(255, 255, 255, 0.5)",
        "light": "rgba(0, 0, 0, 0.15)",
      },
      "width": "2px",
    },
    "check": Object {
      "radius": "100%",
    },
    "gap": "small",
    "hover": Object {
      "border": Object {
        "color": Object {
          "dark": "white",
          "light": "black",
        },
      },
    },
    "icon": Object {},
    "icons": Object {},
    "size": "24px",
  },
  "rangeInput": Object {
    "thumb": Object {},
    "track": Object {
      "color": Array [
        null,
        ";",
      ],
      "height": "4px",
    },
  },
  "rangeSelector": Object {
    "background": Object {
      "invert": Object {
        "color": "light-4",
      },
    },
  },
  "select": Object {
    "container": Object {},
    "control": Object {},
    "icons": Object {
      "margin": Object {
        "horizontal": "small",
      },
    },
    "options": Object {
      "container": Object {
        "align": "start",
        "pad": "small",
      },
      "text": Object {
        "margin": "none",
      },
    },
    "step": 20,
  },
  "tab": Object {
    "active": Object {
      "color": "text",
    },
    "border": Object {
      "active": Object {
        "color": Object {
          "dark": "white",
          "light": "black",
        },
      },
      "color": Object {
        "dark": "accent-1",
        "light": "brand",
      },
      "hover": Object {
        "color": Object {
          "dark": "white",
          "light": "black",
        },
      },
      "side": "bottom",
      "size": "small",
    },
    "color": "control",
    "hover": Object {
      "color": Object {
        "dark": "white",
        "light": "black",
      },
    },
    "margin": Object {
      "horizontal": "small",
      "vertical": "xxsmall",
    },
    "pad": Object {
      "bottom": "xsmall",
    },
  },
  "table": Object {
    "body": Object {
      "align": "start",
      "pad": Object {
        "horizontal": "small",
        "vertical": "xsmall",
      },
    },
    "footer": Object {
      "align": "start",
      "border": "top",
      "pad": Object {
        "horizontal": "small",
        "vertical": "xsmall",
      },
    },
    "header": Object {
      "align": "start",
      "border": "bottom",
      "pad": Object {
        "horizontal": "small",
        "vertical": "xsmall",
      },
    },
  },
  "tabs": Object {
    "header": Object {},
    "panel": Object {},
  },
  "text": Object {
    "large": Object {
      "height": "28px",
      "maxWidth": "528px",
      "size": "22px",
    },
    "medium": Object {
      "height": "24px",
      "maxWidth": "432px",
      "size": "18px",
    },
    "small": Object {
      "height": "20px",
      "maxWidth": "336px",
      "size": "14px",
    },
    "xlarge": Object {
      "height": "32px",
      "maxWidth": "624px",
      "size": "26px",
    },
    "xsmall": Object {
      "height": "18px",
      "maxWidth": "288px",
      "size": "12px",
    },
    "xxlarge": Object {
      "height": "40px",
      "maxWidth": "816px",
      "size": "34px",
    },
  },
  "textArea": Object {},
  "textInput": Object {},
  "video": Object {
    "captions": Object {
      "background": "rgba(0, 0, 0, 0.7)",
    },
    "icons": Object {},
    "scrubber": Object {
      "color": "light-4",
    },
  },
  "worldMap": Object {
    "color": "light-3",
    "continent": Object {
      "active": "8px",
      "base": "6px",
    },
    "hover": Object {
      "color": "light-4",
    },
    "place": Object {
      "active": "20px",
      "base": "8px",
    },
  },
}
`;
